### DISCUSSION TITLE
Some personality types and moods for villagers aren't available in the most recent versions of the mod.

### DISCUSSION THREAD

**Discussion by NcrRanger90** [Luke100000/minecraft-comes-alive#910] [General]
First of all, i don't want to sound ungrateful or come across rude. I am very grateful for this awesome free mod. I am just wandering if there were any technical constraints or maybe creative decisions that led to removing a lot of personality types and moods from the villagers. I switched from 1.18.2 to 1.20.1, and in the older version of the mod, villagers had more moods and also more personalities, which made the mod more immersive.

**Comment by Luke100000** [comment]
I think the general idea was to reduce personalities, and then focus on building on top of them and integrating them more. For example, until now, we had many different moods, but they were not used. Just fancy names without any meaning. @JahxSenpoopie  might add more details here.

**Reply by NcrRanger90** [reply]
Thanks for replying. 
Quality over quantity. It makes sense.
